packages/mux-player/src/index.ts

```diff
@@ -1,6 +1,5 @@
 import "media-chrome";
 import "@mux-elements/mux-video";
-import MxpDialog from "./dialog";
```

Comment:
@luwes we have this import in the template, so I don't think we need it here. I think it was getting removed during the build process